analysis/src/CompletionBackEnd.ml

```diff
@@ -1455,6 +1455,131 @@ let getOpens ~debug ~rawOpens ~package ~env =
   (* Last open takes priority *)
   List.rev resolvedOpens
 
+let getArgs ~env (t : Types.type_expr) ~full =
```

Comment:
This one is mostly copied from a pre-existing fn, but with identifying what type of argument is found (labelled or unlabelled).